tests/Microsoft.Identity.Test.Unit/CDT/CdtTests.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-#if PREVIEW_CDT
+#if NET6_0_OR_GREATER
```

Comment:
Do we not expect things to work on .NET Fwk? Id.Web supports netstandard and netfwk too?

Reply:
It should, I was running into some weird .net issues on my machine because of mismatched packages. Will remove soon